projects/components/src/icon/icon-size.ts

```diff
@@ -4,6 +4,7 @@ export const enum IconSize {
   Medium = 'medium',
   Large = 'large',
   ExtraLarge = 'extra-large',
+  ExtraLarger = 'extra-larger',
```

Comment:
We need better names

Reply:
Oh, and here I was hoping to squeeze in an `ExtraMoreLargerest` at some point.